### ISSUE TITLE
Git Dependency

### ISSUE THREAD

**Issue by ForbesLindesay** [karthikv/nodefront#18]
I see we're using: git://github.com/karthikv/commander.js.git as our commander dependency rather than depending on the npm package.  I assume this is just because of [your pull request](https://github.com/visionmedia/commander.js/pull/75)?

Is that issue urgent enough that we can't wait for a pull request to be accepted into commander.js ?  The git dependency makes us require git to be installed on the client machine (in addition to node.js).  I'd like to minimise the dependencies required to install our tool as it'll really hurt adoption.  I tried to install this on a friends computer and it failed because of the git dependency.

I do think the issue needs fixing, but I think you'll probably need to submit a new pull request for it that builds on the work already done and fixes it in a single commit.

Also, a single pull request should really fix a single thing, so submit a second pull request for the issue with the test as it's completely unrelated and @visionmedia should be able to accept or decline them separately.

**Comment by karthikv** [comment]
It's quite an important issue, as I tend to like placing my command-line options upfront. I think users will find it unintuitive if they are not allowed to do so. Nevertheless, the severity has gone down due to the recent fix that somewhat solves the problem, so I guess we can switch back to the normal commander dependency. I'll update package.json later today.

In terms of the pull request, I've narrowed it down to one commit that focuses on the main issue I mentioned.

As a result of this change, running `nodefront` with no options won't print out help anymore, because that relied on a change I had made to my fork of commander.js. I'll also need to fix up this problem.
